content/blogs/2024-02-06-gitops.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "GitOps with Kestra: Beyond the Trend"
+description: "GitOps is a keystone when working with automated workflows"
+date: 2024-02-06T10:00:00
+category: Solutions
+author:
+  name: Benoit Pimpaud
+  image: "bpimpaud"
+image: /blogs/2024-02-06-gitops.png
+---
+
+Recent trends all mention Ops as a shiny suffix. Where did that come from? Is it pure marketing or is there something real behind it?
+As a basic definition, GitOps is the paradigm making systems reproducible based on the state of a Git repository.
+Being reproducible based on the state of a Git repository embeds the real power of software engineering.
+
+As Max Beauchemin said: “There’s a multitude of reasons why complex pieces of software are not developed using drag and drop tools: it’s that ultimately code is the best abstraction there is for software”.
+
+[If you can’t express what you want in words, no amount of clicking will get you the right answer. Moreover, if you can express it in words, it’s not a big leap to express it in code.](https://twitter.com/jsylvest/status/1703207785335288086)
+
+As software engineers, we already know about distributed version control systems. We use Git every day.
+Adding “Ops” here is the realization that consistency, efficiency, and scaling practices are hard to achieve without a proper source of truth and automation processes. In some higher gazes, it extends beyond Git and the software itself.
+
+It’s unsurprising to see every part of the company adding Ops as a suffix in job positions: DataOps, SalesOps, CRMOps, RevOps, etc. Mastering operations is all about adding automated processes and peer-reviewing methods. Hence the trends around “Ops”.
+
+While creating such processes is mostly a human policy matter, our tools still should make this as clear as possible and give us good guidelines.
+This is the goal of this blog post regarding Kestra alongside Git.
+
+
+## Kestra: All As Code and All From the UI
+
+One key point about enabling good development and operation processes is the interface. Our vision at Kestra is to let everyone choose their preferences. As Kestra is fully API-based, it’s easy to expose the Kestra system to different interfaces: Kestra User Interface is the most obvious one but users could also use VS Code extension to develop flows, use Terraform provider to interact with Kestra’s API, use the API directly, etc.
+This openness is very important to let everyone find their spot. For both development and operation.
+
+You might want to use code to define your assets at some maturity level. Having one single source of truth from which we can deploy, review, and edit those assets is critical.
+
+This is why every developer uses Git nowadays.
+That’s why we introduced new Git tasks and capabilities in Kestra.
```

Comment:
```suggestion
```

I'd cut this part as we don't operate on asset level + duplication of what was said. Ending the paragraph with this sentence "This openness is essential to let everyone find their spot — for both development and operation." makes the paragraph stronger.

Reply:
done